usaspending_api/etl/tests/test_es_rapidloader.py

```diff
@@ -1,4 +1,5 @@
 import pytest
```

Comment:
maybe change the name of this test module to `test_elasticsearch_indexer.py`?

Reply:
To be fair there tests aren't truly testing that management command either.